### PR TITLE
Postpone ChronoMint code initialization until first render

### PR DESCRIPTION
**Pull request by olegdizus**
- ChronoMint initialization errors breaks JS execution and prevent to render any meaningful info.
- App crashes without any warning to user
- App can't send crash info
- Solving problem make app startup much faster

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -28,7 +28,6 @@
 node_modules/react-native/Libraries/react-native/react-native-interface.js
 node_modules/react-native/flow/
 node_modules/react-native/flow-github/
-node_modules/immutable/dist/immutable.js.flow
```
Comment (ozalexo) [inline .flowconfig]: [MINOR] Q: Why it was deleted?

File: src/redux/ducks.js [hunk 2]
```diff
@@ -47,6 +47,6 @@ export {
   watcher,
 }
 
-export type TState = {
+export type TState = {} | {
```
Comment (ozalexo) [inline src/redux/ducks.js]: Q: Maybe optional 'sensitive' will be better?
```
export type TState ={
   sensitive?: TStateSensitive
 }
```

Comment (olegdizus) [reply src/redux/ducks.js]: - TState will contain all state fields in future
- On start state is empty object, so it type is correct

File: .flowconfig (hunk 1 above)
Comment (olegdizus) [reply .flowconfig]: It is descripted in commit message:

Prevent flow memory leak

It is not fixing problem with immutable types,
but now flow is not consuming RAM so much.

**Pull request merged**